SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -6874,6 +6874,12 @@ void CEulerSolver::BC_Sym_Plane(CGeometry      *geometry,
             velocity in normal direction: v_r = v - 2 (v*n)n ---*/
       ProjVelocity_i = nodes->GetProjVel(iPoint,UnitNormal);
 
+      /*--- Adjustment to v.n due to grid movement. ---*/
+      if (dynamic_grid) {
+          for (iDim = 0; iDim < nDim; iDim++)
+            ProjVelocity_i -= geometry->nodes->GetGridVel(iPoint)[iDim]*UnitNormal[iDim];
+      }
```

Comment:
I added a geometry toolbox that defines dot products and other functions, I think it makes these bits of code more expressive.

Reply:
Nice, we should use those as much as possible. Will help minimize bugs.

My only complaint is the long namespace name.. maybe we could standardize on some acronyms like AD, GT, PT, etc. Just a thought

Reply:
Had to look it up, it is possible to alias namespaces (https://en.cppreference.com/w/cpp/language/namespace_alias).